blocks/library/block/index.js

```diff
@@ -158,8 +158,8 @@ const ConnectedReusableBlockEdit = connect(
 export const name = 'core/block';
 
 export const settings = {
-	title: __( 'Reusable Block' ),
-	category: 'reusable-blocks',
+	title: __( 'Shared Block' ),
+	category: 'shared',
```

Comment:
It's a bit odd that we assign a category here, since the block never appears on its own in an inserter. But I guess with our block validation, it's not possible to _not_ assign a category. Maybe we should take `isPrivate` into consideration when determining whether a category is needed.

__Edit:__ Might also impact blocks which are expected to _only_ be used in a nested context.
__Edit 2:__ We could just make `category` non-required, though it's easy to mistakenly omit and causes a block to not appear in the inserter.

Reply:
I think making category non-required might be the best in any case.

Reply:
I would double check if the shared block won’t show up in the inserter with those changes introduced.

I second making category optional and to be expressed as any alphanumerical string. In the inserter we would put it under others category if it isn’t one of those that core provides.
In general I think we should convert shared block logic into the editor UI instead of introducing isPrivate property.

Reply:
I think the last time the issue was with the recent blocks in both the regular inserter and in the inline inserter (right hand icons).